packages/ui/src/components/Popup/helpers.ts

```diff
@@ -73,6 +73,34 @@ const computePlacement = ({
   return 'top'
 }
 
+/**
+ * This function will check if the offset parent is usable for popup positioning
+ * If not it will loop and search for a compatible parent until document.body is reached
+ */
+const findOffsetParent = (element: RefObject<HTMLDivElement>) => {
+  const offsetParent = element?.current?.offsetParent
+
+  if (offsetParent && !['TH', 'TD', 'TABLE'].includes(offsetParent.tagName)) {
+    return offsetParent
+  }
+
+  let currentElement = element?.current
+
+  while (currentElement && currentElement.tagName !== 'BODY') {
+    const { position } = window.getComputedStyle(currentElement)
```

Comment:
check if window exist ? it's will not work server side ?

Reply:
@philibea should exists as this function is only executed in useEffect, the build of examples seems fine so I think it's okay. Wdyt?